packages/devtools-components/src/tree.css

```diff
@@ -75,6 +75,7 @@
   transition: transform 0.125s ease;
   width: 10px;
   margin-inline-end: 5px;
+  -webkit-margin-end: 5px;
```

Comment:
we shouldnt need -webkit because we're using postcss autoprefix

Reply:
so, if the debugger is using postCSS, what's the issue here ?

Reply:
Doing some more testing (hopefully this is useful):

If I add a `postcss.config.js` file:  

```js
module.exports = ({ file, options, env }) => {
  return {
    plugins: [
      require("autoprefixer")({
        browsers: ["last 2 Firefox versions", "last 2 Chrome versions"],
        flexbox: false,
        grid: false
      })
    ]
  };
};
```

into `debugger.html/node_modules/devtools-components/src` and then delete the  
`debugger.html/node_modules/devtools-components/postcss.config.js` and restart the app, PostCSS does indeed add the -webkit prefix.

Reply:
so, it's about the config in the debugger not targeting CSS in node_modules right ?
Could this be resolved on the debugger side ?